imports/services/isEditedService.js

```diff
@@ -0,0 +1,279 @@
+import { Meteor } from 'meteor/meteor';
+import '/imports/helpers/promisedMethods';
+
+import { MinutesSchema } from '../collections/minutes.schema';
+import { Minutes } from '../minutes';
+import { MeetingSeriesSchema } from '../collections/meetingseries.schema';
+import { MeetingSeries } from '../meetingseries';
+import {Topic} from '../topic';
+
+function setIsEditedMS(msId) {
+    let ms = new MeetingSeries(msId);
+
+    ms.isEditedBy = Meteor.userId();
+    ms.isEditedDate = new Date();
+
+    ms.save();
+}
+
+function removeIsEditedMS(msId, any) {
+    let unset = false;
+    let ms = new MeetingSeries(msId);
+
+    if (any === true) {
+        unset = true;
+    }
+    else {
+        if (ms.isEditedBy === Meteor.userId()) {
+            unset = true;
+        }
+    }
+
+    if (unset === true) {
+        ms.isEditedBy = null;
+        ms.isEditedDate = null;
+        ms.save();
+    }
+}
+
+function removeIsEditedMin(minuteId, any) {
```

Comment:
Could you please rename the parameter `any` so it explains more directly what the parameter is used for? The same applies for the `any`-parameter used in the other functions. Thanks!